mlflow/tracking/_tracking_service/client.py

```diff
@@ -827,6 +827,12 @@ def create_logged_model(
             model_type=model_type,
         )
 
+    def log_logged_model_params(self, model_id: str, params: dict[str, str]) -> None:
```

Comment:
```suggestion
    def log_model_params(self, model_id: str, params: dict[str, str]) -> None:
```
Let's be consistent everywhere :)